apache-rat-core/src/main/java/org/apache/rat/report/claim/impl/xml/SimpleXmlClaimReporter.java

```diff
@@ -140,9 +100,9 @@ public void startReport() throws RatException {
     @Override
     public void endReport() throws RatException {
         try {
-            writer.closeDocument();
+            writer.closeElement();
         } catch (IOException e) {
-            throw new RatException("Cannot close last element", e);
+            throw new RatException("Cannot open start element", e);
```

Comment:
"Cannot close start element"? endReport() should not open the start element here.